app-utils/gum/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=gum
+PKGSEC=utils
+PKGDES="Tool for glamorous shell scripts"
```

Comment:
What does this tool do? Remove extraneous descriptive words like "glamorous", describe what's the package for. Follow https://wiki.aosc.io/developer/packaging/package-styling-manual/ .